experimental/segmenter/src/lstm.rs

```diff
@@ -86,8 +166,25 @@ impl LstmSegmenterIteratorUtf16 {
 
 #[cfg(test)]
 pub fn get_line_break_utf8(input: &str) -> Option<Vec<usize>> {
-    let iter = LstmSegmenterIterator::new(&*THAI_LSTM, input);
-    let result: Vec<usize> = iter.collect();
+    let mut result: Vec<usize> = Vec::new();
+    let mut lang_iter = LanguageIterator::new(&input);
+    let mut offset = 0;
+    loop {
+        let str_per_lang = lang_iter.next();
+        if str_per_lang.is_none() {
+            break;
+        }
```

Comment:
Suggestion: we can replace these lines and line 181 with `for str_per_lang in lang_iter`, right?